finagle-http/src/main/scala/com/twitter/finagle/http/filter/DtabFilter.scala

```diff
@@ -40,7 +47,45 @@ object DtabFilter {
     Future.value(rsp)
   }
 
+  @deprecated("use DtabFilter.Decoder", "2016-06-07")
   class Finagle[Req <: Message] extends DtabFilter[Req, Response] {
     def respondToInvalid(req: Req, msg: String) = invalidResponse(msg)
   }
+
+  /**
+   * Extracts Dtab-local headers from incoming requests and adds the
+   * Dtab to the local context.
+   */
+  class Extractor extends DtabFilter[Request, Response] {
+    def respondToInvalid(req: Request, msg: String) = invalidResponse(msg)
+  }
+
+  /**
+   * Modifies each request with Dtab encoding from Dtab.local and
+   * streams chunked responses via `Reader`.  If the request already
+   * contains Dtab headers they will be dropped silently.
+   */
+  class Injector extends SimpleFilter[Request, Response] {
+    private[this] val hasSetDtab = Request.Schema.newField[Boolean](false)
```

Comment:
since this is a constant it should be capitalized and go into the companion object.